sylt-compiler/src/dependency.rs

```diff
@@ -234,14 +236,15 @@ fn order(
                 State::Inserted => Ok(()),
             },
         };
-        let (deps, statement) = to_order.get(&name).unwrap();
-        for dep in deps {
-            recurse(*dep, to_order, inserted, ordered)
-                .map_err(|mut cycle| { cycle.push(*statement); cycle })?;
+        if let Some((deps, statement)) = to_order.get(&name) {
+            for dep in deps {
+                recurse(*dep, to_order, inserted, ordered)
+                    .map_err(|mut cycle| { cycle.push(*statement); cycle })?;
+            }
+            ordered.push(*statement);
```

Comment:
Did this need to change? It should definitely crash if the get fails since `name` is originally taken out of `to_order`. That is a logic error otherwise.